### PR TITLE
multiple rewrites and fixes

### PR DESCRIPTION
**Pull request by Leybl**
removed 2017 date and rewrote section to emphasize cost advantages (see line 49).

### CODE REVIEW

**Comment by PRMerger20** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/netfx/kusto-ingest-best-practices.md [hunk 1]
```diff
@@ -12,43 +12,54 @@ ms.date: 08/16/2019
 # Kusto Ingest Client Library - Best Practices
 
 ## Choosing the right IngestClient flavor
+
 Using [KustoQueuedIngestClient](kusto-ingest-client-reference.md#interface-ikustoqueuedingestclient) is the recommended native data ingestion mode. Here's why:
-* Direct ingestion is impossible during Kusto Engine downtime (e.g. during deployment), while in the Queued ingestion mode the requests are persisted to Azure queue and the Data Management service will retry as needed.
-* Data Management service is responsible to not overload the engine with ingestion requests. Overriding this control (e.g. using Direct ingestion) might severely affect engine performance, both ingestion and query.
+* Direct ingestion is impossible during engine downtime (e.g. during deployment), whereas in the queued ingestion mode the requests are persisted to the Azure queue, and the Data Management service will retry as needed.
+* The Data Management service is responsible to not overload the ngine with ingestion requests. Overriding this control (e.g. using Direct ingestion) might severely affect engine performance, both ingestion and query.
```
Comment (orspod) [inline data-explorer/kusto/api/netfx/kusto-ingest-best-practices.md:18]:
```suggestion
* The Data Management service is responsible to not overload the engine with ingestion requests. Overriding this control (e.g. using Direct ingestion) might severely affect engine performance, both ingestion and query.
```

**Comment by PRMerger12** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**